extensions/sql-migration/src/dialog/skuRecommendationResults/skuRecommendationResultsDialog.ts

```diff
@@ -303,7 +303,10 @@ export class SkuRecommendationResultsDialog {
 			{ value: constants.SQL_TEMPDB },
 			{
 				value: recommendation.targetSku.tempDbDiskSizes!.length > 0
-					? constants.STORAGE_CONFIGURATION(recommendation.targetSku.logDiskSizes![0].size, recommendation.targetSku.logDiskSizes!.length)
+					? constants.STORAGE_CONFIGURATION(
+						recommendation.targetSku.logDiskSizes!.length,
+						this.getDiskTypeText(recommendation.targetSku.logDiskSizes![0].type),
```

Comment:
This invariant that all disk types are same, might break in future, so might be better to take a count based on the size :)

Reply:
Done.